src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -26,14 +26,13 @@ public class AddCommand extends Command {
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
-            + "[" + PREFIX_TAG + "TAG]... "
-            + "[" + PREFIX_TAG + "COMMENT]...\n"
+            + PREFIX_TELEGRAM_HANDLE + "ADDRESS "
```

Comment:
The telegram handle is no longer referred to as an address - the help message should appropriately reflect this, similar to the edit command.